pages.fr/common/csh.md

```diff
@@ -0,0 +1,21 @@
+# csh
+
+> Le shell (interprèteur de commandes) affiche une syntaxe proche de C.
+> Voir aussi : tcsh.
+> Plus d'informations : https://www.mkssoftware.com/docs/man1/csh.1.asp.
```

Comment:
```suggestion
> Plus d'informations : <https://www.mkssoftware.com/docs/man1/csh.1.asp>.
```
We wrap our More information links inside `<>` (angular brackets). Once this suggestion is resolved the PR is GTG.